AirCasting/Utils/DataFormatters.swift

```diff
@@ -116,6 +116,14 @@ enum DateFormatters {
             return dateFormatter
         }()
     }
+    
+    enum Debug {
+        static let logsFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.dateFormat = "yyyy-MM-dd HH:mm:ss"
```

Comment:
Can we display this time in UTC so that it is consistent with what we store in db?